scripts/ovpsim_log_to_trace_csv.py

```diff
@@ -268,6 +272,14 @@ def process_ovpsim_sim_log(ovpsim_log, csv, full_trace = 1, stop = 0,
         show_line_instr(line, prev_trace)
 
         if full_trace:
+            # TODO - when got full ins decode remove this
+            if  0 or "fsriw" in line or \
```

Comment:
"0 or" seems to be redundant

Reply:
yes - but it is a work in progress and will be fixed in due course

Reply:
do I just push it to my repo and the pull will be notified?

Reply:
git add .
git commit --amend
git push origin HEAD -f